File: pyg_manager_web/src/main/webapp/js/service/specificationService.js
//自定义服务
app.service('specificationService',function ($http) {
    ////定义查询所有品牌列表的方法
    this.findAll=function () {
        return $http.get("../specification/findAll.do");
    }
    //条件查询和分页展示
    this.search=function (searchEntity,pageNum,pageSize) {
        return $http.post("../specification/search.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
    }
    //添加和修改
    this.add=function (entity) {
        return $http.post("../specification/add.do",entity);
    }
    //修改
    this.update=function (entity) {
        return $http.post("../specification/update.do",entity);
    }
    //通过id查找
    this.findOne=function (id) {
        return $http.get("../specification/findOne.do?id="+id);
    }
    //删除
    this.delete=function (selectIds) {
        return $http.get("../specification/delete.do?ids="+selectIds);
    }


})

File: pyg_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.enity.PageResult;
import com.pinyougou.groupEntity.Specification;
import com.pinyougou.pojo.TbSpecification;

public interface SpecificationService {
   //分页+条件查询
    PageResult findPage(TbSpecification specification, Integer pageNum, Integer pageSize);
    //新增
   void add(Specification specification);
    //查找根据id
    Specification findOne(Long id);
    //修改
    void update(Specification specification);
    //删除
    void delete(Long[] ids);
}
